Views Milestone

AyoubM490 opened the pull request:
In this commit, I added the views for:
- posts, and
- users

AyoubM490 committed:
In this commit, I added the views for:
- posts, and
- users

AyoubM490 committed:
add stylelint and fix linter error

dasileker reviewed (changes requested):
### **Hi @AyoubMs,** 

### **Good job so far!**
There are some issues that you still need to work on to go to the next project but you are almost there!

## **Highlights**

- everything looks cool.


## Required Changes ♻️
_Check the comments under the review._

- **Hello and nice job on the blog app, but there are some minor issues that need to be solved**
- **1 there is some issue in the migration files like `ChangeForeignKeyForComments and ChangeForeignKeyForPosts and ChangeForeignKeyForLikes` the user_id and the author_id is not supposed to be changed and that causes an error and for that, they can not be stored in the database**
- **2 the `routes` are having something different, and for that reason, the data stored is not displayed**

![Capture d’écran 2022-07-13 212343](https://user-images.githubusercontent.com/61562626/178827789-e805d27d-8f16-47b0-8a6d-b7a670c771ed.png)


![Capture d’écran 2022-07-13 211200](https://user-images.githubusercontent.com/61562626/178827279-7a9f9540-e513-418e-925d-c39891ab350f.png)

![Capture d’écran 2022-07-13 212420](https://user-images.githubusercontent.com/61562626/178827874-d542d009-c8ad-4cb1-85cb-0db8c9bc9b8a.png)


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AyoubM490 committed:
In this commit, I removed migration files for changing user_id to author_id,
and that caused a change in the schema.rb file, and all other related files
such as the models files.
I changed the tests files, and runned the tests, and they succeeded.
However, when I tried to add users from the console using the "Processing data
in models" data, it failed, because it needs emails, passwords and changing
author to user. Please consider that when you are testing the app.
Thank you

AyoubM490 committed:
fix rubocop linter errors

sja-thedude reviewed (changes requested):
## Required Changes ♻️

Hi @AyoubMs, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![200w](https://user-images.githubusercontent.com/70269507/159484303-3187dacc-4a82-4263-bb1a-8d72417ce1d9.gif)

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
- [x] As suggested previously, kindly make sure your app works as per the wireframe. When I tried to add a user and then run User.all, I cant see any user added there nor on the app. Make sure to fix it.
<img width="1728" alt="Screenshot 2022-07-14 at 3 14 38 PM" src="https://user-images.githubusercontent.com/70269507/178956971-cd5a7458-815e-4b1d-a877-0806e1335c68.png">



## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AyoubM490 committed:
In this commit, I removed devise, and bugs related to changing user column
with author column

AyoubM490 committed:
fix rubocop linter error

gemmen29 reviewed (approved):
Hi @AyoubMs, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

![](https://media.giphy.com/media/mCIjCgs3nWQWfJZvPA/giphy-downsized-large.gif)

## Optional suggestions

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
